runtime/syntax/cs.vim

```diff
@@ -1,10 +1,11 @@
 " Vim syntax file
-" Language:	C#
-" Maintainer:	Anduin Withers <awithers@anduin.com>
-" Former Maintainer:	Johannes Zellner <johannes@zellner.org>
-" Last Change:	Fri Aug 14 13:56:37 PDT 2009
-" Filenames:	*.cs
-" $Id: cs.vim,v 1.4 2006/05/03 21:20:02 vimboss Exp $
+" Language:		C#
+" Maintainer:		Nick Jensen <nickspoon@gmail.com>
+" Former Maintainer:	Anduin Withers <awithers@anduin.com>
```

Comment:
You removed Johannes' name but it is not good.
You should change "Former Maintainer" to "Former Maintainers" and keep Johannes' name.

Reply:
Hi @k-takata,

Yes Bram said the same thing in an email. He has added Johannes back in to the current version [here](https://github.com/vim/vim/blob/master/runtime/syntax/cs.vim), and I have updated my repo [here](https://github.com/nickspoons/vim-cs/blob/master/syntax/cs.vim)

Reply:
Oh, sorry. I missed that.